microcosm_pubsub/consumer.py

```diff
@@ -5,7 +5,7 @@
 from os.path import exists
 from urllib.parse import urlparse
 
-from boto3 import Session
+from boto3.session import Session
```

Comment:
Why are both of these valid? Why change it?

Reply:
Session is imported into `boto3.__init__.py` so it works. It's not part of the typing stubs, though, so mypy complains about the indirect import.  I guess this is due to newer stubs or newer mypy since the last changes to this library.